source/testing/test-helpers.md

```diff
@@ -37,7 +37,7 @@ next one starts.
     is complete.
 * `fillIn(selector, text)`
   - Fills in the selected input with the given text and returns a promise that
-     fulfills when all resulting async behavior is complete. Works with `<select>` elements as well as `<input>` elements.
+     fulfills when all resulting async behavior is complete. Works with `<select>` elements as well as `<input>` elements (with select elements, fill in the value of the option, e.g. 'true' and not the display, e.g. 'Yes').
```

Comment:
Well with input they're the same, right, so that's just straightforward. I just recently noticed this "gotcha" though with selects since I was trying to fill in with the value the user would see, and couldn't understand why it wasn't working all of a sudden, since my others had worked fine (the selects where the value and display were the same).